### PR TITLE
[WFLY-15879] Exclude the log4j-api JAR from getting transformed.

### PR DESCRIPTION
**Pull request by jamezp**
https://issues.redhat.com/browse/WFLY-15879

### CODE REVIEW

**Review by darranl — APPROVED** [review]

File: ee-9/feature-pack/pom.xml [hunk 1]
```diff
@@ -481,6 +481,9 @@
                                      functionality (e.g. consoles or similar) that we do not support. -->
                                 <exclude>com.h2database:h2\z</exclude>
                                 <exclude>org.apache.thrift:libthrift\z</exclude>
+                                <!-- Only contains hard-coded strings that are either not used or both the javax and
+                                jakarta names are used -->
+                                <exclude>org.apache.logging.log4j:log4j-api\z</exclude>
```
Comment (bstansberry) [inline ee-9/feature-pack/pom.xml:486]: @jamezp Please move this down. These are not strictly alphabetical; they are grouped by the explanatory comment so this PR now separates jasypt from its explanation.

**Review by bstansberry — CHANGES_REQUESTED** [review]
I'm requesting a change. Please feel free to merge this once it's made; i.e. don't block on me confirming.

File: ee-9/feature-pack/pom.xml (hunk 1 above)
Comment (jamezp) [reply ee-9/feature-pack/pom.xml]: Done. Moved to the bottom of the exclusions as it didn't seem to fit in the other categories.

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks, @jamezp

**Pull request merged**